compiler/plugins/target/AMD-AIE/aievec/VectorToVectorConversions.cpp

```diff
@@ -16,15 +16,20 @@
 #include "AIEVecUtils.h"
 #include "Passes.h"
 #include "Utils.h"
+#include "aie/Dialect/AIEVec/AIEVecUtils.h"
```

Comment:
These are mlir-aie headers. We should not use absolutely any mlir-aie headers or code directly. I "ported" all the relevant stuff from them into `Utils.h` that's in this same dir.